packages/block-library/src/post-excerpt/block.json

```diff
@@ -2,6 +2,9 @@
 	"name": "core/post-excerpt",
 	"category": "design",
 	"attributes": {
+		"align": {
```

Comment:
In my opinion, this should most likely not be called `align`, but rather `textAlign`. I know `align` is how the Paragraph block does it, but this is confusing since there is also block alignment, which (if using the `supports` flag) uses `align` as its attribute name.

Reply:
Agreed, `textAlign` or `alignText` would be preferable.  I saw the `align` support hook is for block alignment as well and noticed the confusion.  I wonder how many other blocks have this conflicted naming convention, but we should start updating them.

Reply:
Or would it be better to similarly update the `align` hook to be `blockAlign` or something similar?

If this is more recent than some of the blocks that use `align` for textAlign, then less people should be effected by the change.  That is, Im assuming if we updated the paragraph block's attr name then a user who updates their Gutenberg version will have whatever alignment settings they saved lost?

Reply:
Is there a compatibility layer for changing a block attribute name? I think there is something for updating a block to a newer version without loosing settings 🤔

Reply:
> Is there a compatibility layer for changing a block attribute name? I think there is something for updating a block to a newer version without loosing settings 🤔

That would be good, at the very least for now I will update the FSE blocks that are using `align` for text alignment to use `textAlign`

Reply:
There is indeed a way to easily handle (what would otherwise be backward-incompatible) changes in block attribute schemas:

https://developer.wordpress.org/block-editor/developers/block-api/block-deprecation/

A lot of core blocks already make use this, storing their deprecated versions in a `deprecated.js` file.

Reply:
Updated for the FSE blocks specifically - https://github.com/WordPress/gutenberg/pull/24077